### DISCUSSION TITLE
Mannequins not working in discussions

### DISCUSSION THREAD

**Discussion by lukens** [community/community#53116] [Discussions] [answered]
### Select Topic Area

Bug

### Body

It seems that mannequins aren't working in discussions. I migrated a project from enterprise server to enterprise cloud, with discussions being one of the features we were keen to use.

After migrating I promoted a number of "issues" from our cloud version too "discussions" in the cloud version.

However, after adding the rest of our team, and having them claim their mannequins, they are still appearing in discussions as their old usernames.

The exception is myself, which I think is potentially because I claimed my mannequin before moving the issues to discussions.

Is this a bug? by design? can I resolve this at all?

**Comment by lukens** [comment]
The original issue the discussion was created from is now showing the correct users, but the discussion is not.

As the discussions haven't progressed at all since creating them, I wondered if I could recreate them from the issues again, but it doesn't look like this is possible.

**Reply by lukens** [reply]
I found that if I deleted the discussion, I could then recreate it from the issue again. Users were correctly attributed.

Sadly not all our users are active on the cloud instance yet, so not all mannequins have yet been claimed, so this will still be an issue.

I'm assuming the rationale is that discussions aren't available on GHES, and so there would be no point checking issues for mannequins that need to be updated. However, as shown in my case, as discussions can be generated from issues, this can still be a requirement to match mannequins in discussions.

**Reply by jricciardi** [reply, marked as answer]
Hi @lukens! I've brought this to the Discussions team's attention, and we've logged this as a bug 🐞 . It looks like for now, the workaround you identified appears to be the best workaround for this particular situation.

For future users who find this discussion: until a better fix is implemented, all the mannequins should be claimed before converting issues to discussions to avoid this bug. 

And for future enterprise users looking at this: Please add your voice, emojis/upvotes, and any additional context! The more feedback we get from users, the better 👍
